src/libs/actions/IOU.ts

```diff
@@ -3261,14 +3254,13 @@ function getSendInvoiceInformation(
             comment: trimmedComment,
             created,
             merchant,
-            receipt: receiptObject,
+            receipt,
             category,
             tag,
             taxCode,
             taxAmount,
             billable,
             reimbursable: true,
-            filename,
```

Comment:
I think this one needs to stay for now until all the readers are updated, otherwise, I think all the references to `transaction.filename` are going to break.